spacy/ml/models/textcat.py

```diff
@@ -127,6 +148,9 @@ def build_text_classifier_v2(
     linear_model: Model[List[Doc], Floats2d],
     nO: Optional[int] = None,
 ) -> Model[List[Doc], Floats2d]:
+    # TODO: build the model with _build_parametric_attention_with_residual_nonlinear
+    # in spaCy v4. We don't do this in spaCy v3 to preserve model
+    # compatibility.
```

Comment:
Flagging this, too

Reply:
Great catch, we can make the switch as part of this PR so that it doesn't slip through the cracks. Let me do that now.